Jenkinsfile

```diff
@@ -221,6 +221,24 @@ pipeline {
             }
         }
 
+        stage('Integration Tests') {
+            when { not { buildingTag() } }
+            steps {
+                sh """
+                    cd ${GO_REPO_PATH}/verrazzano-operator
+                    make integ-test
+                    cp coverage.html ${WORKSPACE}
```

Comment:
I thought this code coverage stuff was not being used with integ-test

Reply:
removed